new-lamassu-admin/src/pages/Customers/components/IdDataCard.js

```diff
@@ -46,7 +61,9 @@ const IdDataCard = memo(({ customerData, updateCustomer }) => {
     },
     {
       header: 'Gender',
-      display: R.path(['gender'])(idData),
+      display: formatGender(
+        R.path(['gender'])(idData) ?? R.path(['sex'])(idData)
```

Comment:
Since we're doing this might as well change the BC parser to use gender, also leave a comment here explaning why we need this.

Reply:
Still no comment here.